models/stg_recurly__account_history.sql

```diff
@@ -33,11 +33,11 @@ final as (
         vat_number,
         tax_exempt as is_tax_exempt,
         account_country,
-        created_at,
-        updated_at,
-        deleted_at
-    from
-        fields
+        created_at, 
+        updated_at, 
+        deleted_at 
+    from 
+        fields 
```

Comment:
Can you update this to reflect our style standards.
```suggestion
    from  fields 
```